src/jit/compiler.h

```diff
@@ -3495,6 +3495,10 @@ class Compiler
 
     void fgInline();
 
+    void fgRemoveEmptyFinally();
+
+    void fgCloneFinally();
+
```

Comment:
The code is in this PR. It may be github is hiding flowgraph.cpp changes from you (since they are large).